Build.fsx

```diff
@@ -1,4 +1,4 @@
-﻿#r @"packages/FAKE.4.19.0/tools/FakeLib.dll"
+﻿#r @"packages/FAKE.Core/tools/FakeLib.dll"
```

Comment:
Do you want me to perform the rebase and just separate add/remove as separate commits, or want me to rebase and remove this from PR at all, so it will be fired via separate PR later?